librz/arch/var_global.c

```diff
@@ -85,9 +85,14 @@ RZ_API bool rz_analysis_var_global_add(RzAnalysis *analysis, RZ_NONNULL RzAnalys
  * \param name Global variable name
  * \param type Global variable type
  * \param addr Global variable address
+ * \param file File containing source declaration
+ * \param line Line number of source declaration
+ * \param colum Column position of source declaration
  * \return true if succeed
  */
-RZ_API bool rz_analysis_var_global_create(RzAnalysis *analysis, RZ_NONNULL const char *name, RZ_NONNULL RZ_BORROW RzType *type, ut64 addr) {
+RZ_API bool rz_analysis_var_global_create(RzAnalysis *analysis,
```

Comment:
What about also keeping the old API? Just make something like `*_create_with_sourceline()` and keep the old `*_create()` as it was, it could be a wrapper around `*_create_with_sourceline()`